### PR TITLE
Remove Duplicate Self-Assessment

### PR DESCRIPTION
**Pull request by sckimynwa**
Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:

- There are duplicate Self-assessment in this page so i removed one

image

When you're ready to submit your PR, don't forget to add the `$-presubmit` label.

### CODE REVIEW

**Comment by rachelandrew** [comment]
@sckimynwa can you sign the CLA so we can merge this?

**Review by jpmedley — CHANGES_REQUESTED** [review]
Thank you for submitting.

File: src/site/content/ko/accessible/labels-and-text-alternatives/index.md [hunk 1]
```diff
@@ -220,5 +220,3 @@ Check out <a href="/guide">our guide to web performance</a>.
 확인란 레이블이 올바르게 지정된 경우, 스크린 리더는 요소가 확인란 역할을 하고, 선택된 상태이며, 아래의 VoiceOver 예제와 같이 "홍보 제안을 수신하시겠습니까?"라는 이름을 가진 요소를 보고할 수 있습니다.
 
 <figure>{% Img src="image/admin/WklT2ymrCmceyrGUNizF.png", alt="'홍보 제안을 수신하시겠습니까?'를 나타내는 VoiceOver 텍스트 출력", width="640", height="174" %}<br>{% Assessment 'self-assessment' %}</figure>
-
-{% Assessment 'self-assessment' %}
```
[the remark this answers is not in the document]
Comment (sckimynwa) [reply src/site/content/ko/accessible/labels-and-text-alternatives/index.md:224]: @jpmedley Aha Thanks I'll change that

**Comment by sckimynwa** [comment]
> @sckimynwa can you sign the CLA so we can merge this?

I signed the CLA & change things

File: src/site/content/ko/accessible/labels-and-text-alternatives/index.md [hunk 2]
```diff
@@ -219,6 +219,6 @@ Check out <a href="/guide">our guide to web performance</a>.
 
 확인란 레이블이 올바르게 지정된 경우, 스크린 리더는 요소가 확인란 역할을 하고, 선택된 상태이며, 아래의 VoiceOver 예제와 같이 "홍보 제안을 수신하시겠습니까?"라는 이름을 가진 요소를 보고할 수 있습니다.
 
-<figure>{% Img src="image/admin/WklT2ymrCmceyrGUNizF.png", alt="'홍보 제안을 수신하시겠습니까?'를 나타내는 VoiceOver 텍스트 출력", width="640", height="174" %}<br>{% Assessment 'self-assessment' %}</figure>
+<figure>{% Img src="image/admin/WklT2ymrCmceyrGUNizF.png", alt="'홍보 제안을 수신하시겠습니까?'를 나타내는 VoiceOver 텍스트 출력", width="640", height="174" %}<br></figure>
```
Comment (jpmedley) [inline src/site/content/ko/accessible/labels-and-text-alternatives/index.md:222]:
```suggestion
<figure>{% Img src="image/admin/WklT2ymrCmceyrGUNizF.png", alt="'홍보 제안을 수신하시겠습니까?'를 나타내는 VoiceOver 텍스트 출력", width="640", height="174" %}</figure>
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

**Review by jpmedley — APPROVED** [review]

**Pull request merged**